### ISSUE TITLE
Sorry, but the previous issue shouldn't be closed about mamot.fr

### ISSUE THREAD

**Issue by shakasan** [sgrigson/oliphant#11]
I'm sorry and I will even be more straightforward... are you fucking moron or what ?

It's really like : "Oh I heard some french people talking bad about mamot, so ... bah I blocked it."

Are you serious ? Some of you should go either see a therapist or a go back to school to learn the basics !!

The main reasons why mamot.fr is not well seen is because they are against fascist/nazis and if you were little more open to the world around you, you will know that nazis, fascists are more and more present in many european countries and very active against any people fighting them....

And the biggest reason is that they make a specific focus on the freedom of speech and the rights of people, in total opposition of the arbitrary censorship as this fucking blue bird.... And they consider that any moderation action is a huge ethical exercice ! And it should ! Censor/Block/Ban by an algorithm is by design a total error !!

Some services like Mastodon/Twitter crossposter even blocked mamot.fr just for what ?
Because they are fearing to be blocked by Twitter because it's not aligned with their censorship point of view !

So basically, if you want a place where there is algorithm deciding for anything, including who have the right to say what, just go back to Twitter, because you completely missed the point about Mastodon.

La Quadrature du Net (Mamot.fr) is an association fighting for privacy and the rights of people in the numeric era. They are constantly involved in trials against cities and even the french government. So of course they are annoying for some peoples.....

For the same reason, their previous spokesperson was awaited by the FBI when he came to the USA for a conference, in a totally illegal way by the way.... But yeah, for the US Empire, the democratic stuff is just a details... as we can see all over the world with still a bunch of morons believing it's the truth....

So, spreading stupid list like that is not just "it's my list, blah blah" like you mentioned on your blog, but a clear moral and ethical responsibility. And if it was just your list as you said, then just keep it for you and don't spread it with tools and so on.... Spreading it like this is a clear action against groups of peoples, so this is not politically speaking neutral as you tried to say...

Censorship, Ban, Block, should always be the VERY LAST action to consider, after a real analysis and not, like I read in the previous issue, the first one because : "Ah as admin, .... others said it's bad, so yeah, I block it too, because yeah..., it's rumors, but it should be true... and oh yeah, I'm a lazy admin, don't want to bother with ethical question, freedom of speech and, in fine, democratic questions.... even if it's impacting  1000 times more peoples that are using their account honestly/fairly.... And even it's disrupting the federation model.... Because, yeah, I'm admin but don't want to bother with the responsibility that come with it". Like it's said : "With great power comes great responsibility".

And at the end, you are not different from the fascists you are talking about in your pseudo justification on your blog.

Also, by the way, blocking an instance, means blocking any users on it.... I suppose this is collateral damages ??? Which is acceptable because you are the source of truth and the light guiding people because they are too much stupid to decide by themselves if something is good or not.... ? That can be summarized by " The end justifies the means". Which is the signature of all systems that lead to arbitrary, injustice and dictators....

And as analogy... it's like, oh, there is some nazis in Germany, let's put an embargo and block borders of this country.
NO ! This is not the way to do, never, whatever bullshit justification you can find.

If a user is problematic, you block the user, not the whole instance !!!

So keep your list for you, fork Mastodon like this f*** Trump if you want, but stop spreading your shit.

And blocking Mamot.fr, it's like saying Amnesty International is killing/jailing peoples, Greenpeace is throwing oil in the oceans.....or EFF is against privacy....

Maybe it's time to wake up your mind a bit and move away from your screen....

**Comment by Seirdy** [comment]
I just want to correct some misinformation:

> are you fucking moron or what

Several of us are actually just regular morons.

**Comment by sgrigson** [comment]
Duplicate of #10
